test/activerecord_spanner_adapter/transaction_test.rb

```diff
@@ -42,6 +42,15 @@ def test_commit_options
     assert_equal 1000, commit_options[:max_commit_delay]
   end
 
+  def test_exclude_txn_from_change_streams
```

Comment:
This test does not really appear to test anything relevant. The `exclude_txn_from_change_streams` flag goes into the `BeginTransaction` options, but this test sets that flag after the transaction has started, and dos not verify anything regarding the `BeginTransaction` options.